conformance/setup.go

```diff
@@ -352,6 +494,19 @@ func RunOnlyIfNot(v bool) {
 	}
 }
 
+func Warn(message string) {
+	fmt.Fprintf(colorable.NewColorableStderr(), "\n")
+	// print file:line
+	_, file, line, _ := runtime.Caller(1)
+	warncolor := color.New(color.FgMagenta).FprintfFunc()
+	warncolor(colorable.NewColorableStderr(), formatter.Fi(2, "\nWARNING: "+message))
+	fmt.Fprintf(colorable.NewColorableStderr(), "\n")
+	// print message
+	msgcolor := color.New(color.FgWhite).FprintfFunc()
+	msgcolor(colorable.NewColorableStderr(), formatter.Fi(2, "\n"))
+	msgcolor(colorable.NewColorableStderr(), formatter.Fi(2, "%s:%d\n", file, line))
+}
```

Comment:
@jdolitsky is there any issue generating warnings in the conformance test this way?

Reply:
Rebased and updated after recent pushes.